src/ape/managers/converters.py

```diff
@@ -17,33 +17,107 @@
 
 # NOTE: This utility converter ensures that all bytes args can accept hex too
 class HexConverter(ConverterAPI):
+    """
+    A converter that converts ``str`` to ``HexBytes``.
+    """
+
     def is_convertible(self, value: str) -> bool:
+        """
```

Comment:
One thing I think works is that `abstractmethod`s with docstrings still appear if the subclass does not override them. So in cases where a generic do string would do, it would make sense not to update each subclass as that is overly verbose and unnecessary.

Please check that this is the case, but could reduce a lot of work!

Reply:
- [x] `is_convertible()` now comes from the API class